Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1365,12 +1365,17 @@ def return_error(message, error='', outputs=None):
         :type outputs: ``dict or None``
         :param outputs: the outputs that will be returned to playbook/investigation context (optional)
 
+        :type exception_object: ``Exception or None``
```

Comment:
why not giving the `error` argument accept Exceptions as well, that was the original purpose of it